zulipterminal/config/gruvbox.py

```diff
@@ -0,0 +1,73 @@
+from enum import Enum
+
+from zulipterminal.config.themes import add_color_prop
+
+
+# fmt: off
+class Color(Enum):
```

Comment:
You need to add the url of the source of the color code for Gruvbox for reference. https://github.com/zulip/zulip-terminal/blob/56fd5b27f85f648831ef27c4ddc39c24538d533a/zulipterminal/config/themes.py#L80

Reply:
Oh yes, I'll do that :+1: